webapp/src/Controller/Jury/TeamAffiliationController.php

```diff
@@ -59,6 +60,13 @@ public function indexAction(
             'name' => ['title' => 'name', 'sort' => true, 'default_sort' => true],
         ];
 
+        if ($this->isGranted('ROLE_ADMIN')) {
```

Comment:
Should we maybe make a generalized method of this in the base class? Also for the one below? So that we don't have this repeated html everywhere, if we ever want to change styling or something.

Reply:
Done, please check the last two commits.